movement/watch_faces/complication/discgolf_face.c

```diff
@@ -0,0 +1,319 @@
+#include <stdlib.h>
+#include <string.h>
+#include "discgolf_face.h"                                      // Remember to change number of courses in this file
+#include "watch.h"
+#include "watch_utility.h"
+
+/* 
```

Comment:
Thank you for adding documentation! Would you mind moving this comment block into discgolf_face.h as per [template.h](https://github.com/joeycastillo/Sensor-Watch/blob/main/movement/template/template.h)? Thanks!

Reply:
No problem! 
Comment block has been moved to the header file and I also added the license block to both files.